app/scrapers/__init__.py

```diff
@@ -28,7 +29,8 @@
     't': Youtube(),
     'p': Parsijoo(),
     'm': Mojeek(),
-    'v': Dailymotion()
+    'v': Dailymotion(),
+    'gb': Gigablast()
```

Comment:
Unfortunately, this will not work and the google engine will perform the search instead of the gigablast engine.  You need to have a single letter which is unique in the scrapers dict because the [dumb down code](https://github.com/fossasia/query-server/blob/master/app/server.py#L60) __looks only at the very first letter__ of the scraper name.  This is discussed in #271 and #273

Reply:
even yandex has yd. does it need to change?

Reply:
It's only checking the first letter in that case too.

Reply:
Agreed #271 covers that bug